homeassistant/config.py

```diff
@@ -206,16 +206,34 @@ def _filter_bad_internal_external_urls(conf: dict) -> dict:
 
 
 def _raise_issue_if_historic_currency(hass: HomeAssistant, currency: str) -> None:
-    if currency in HISTORIC_CURRENCIES:
-        ir.async_create_issue(
-            hass,
-            "homeassistant",
-            "historic_currency",
-            is_fixable=False,
-            severity=ir.IssueSeverity.WARNING,
-            translation_key="historic_currency",
-            translation_placeholders={"currency": currency},
-        )
+    if currency not in HISTORIC_CURRENCIES:
+        ir.async_delete_issue(hass, "homeassistant", "historic_currency")
+        return
+
+    ir.async_create_issue(
+        hass,
+        "homeassistant",
+        "historic_currency",
+        is_fixable=False,
+        severity=ir.IssueSeverity.WARNING,
+        translation_key="historic_currency",
+        translation_placeholders={"currency": currency},
+    )
+
+
+def _raise_issue_if_no_country(hass: HomeAssistant, country: str | None) -> None:
+    if country is not None:
+        ir.async_delete_issue(hass, "homeassistant", "country_not_configured")
+        return
+
+    ir.async_create_issue(
+        hass,
+        "homeassistant",
+        "country_not_configured",
```

Comment:
Can/should we link to the documentation here?
Or even better the place in the UI where you set it?

Reply:
Fixed!